public/index.html

```diff
@@ -4,18 +4,18 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="initial-scale=1, maximum-scale=1, user-scalable=no, width=device-width">
     <title>teletext</title>
+
+    <script src="libraries/socket.io.js"></script>
     <script src="libraries/p5.js" type="text/javascript"></script>
     <script src="libraries/p5.dom.js" type="text/javascript"></script>
+
     <script src="ttxpage.js" type="text/javascript"></script>
     <script src="edittf.js" type="text/javascript"></script>
     <script src="charchanged.js" type="text/javascript"></script>
     <script src="sketch.js" type="text/javascript"></script>
     <script src="cursor.js" type="text/javascript"></script>
-    <!-- <script src="https://cdn.socket.io/socket.io-1.4.5.js"></script> -->
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/socket.io/2.1.1/socket.io.js"></script>
-	
-	  <meta name="viewport" content="user-scalable=yes,initial-scale=0.5,maximum-scale=1.5,minimum-scale=0.4,width=display-width">
```

Comment:
This line is resetting the `viewport` meta tag from above, meaning that the one above has no effect.

So I've removed this one in favour of the initial one - however, check which one was actually wanted, since the below one was actually being applied via HTML precedence rules